src/App.tsx

```diff
@@ -53,6 +53,71 @@ function App() {
     { timestamp: number; method: string; data: any }[]
   >([]);
 
+  const setInitialMethodsAndAccounts = (currentSession: any) => {
+    const initialSelectedMethods: Record<string, string> = {};
+    const initialSelectedAccounts: Record<string, CaipAccountId> = {};
+
+    Object.entries(currentSession.sessionScopes).forEach(
+      ([scope, details]: [string, any]) => {
+        initialSelectedMethods[scope] = 'eth_blockNumber';
+
+        if (details.accounts && details.accounts.length > 0) {
+          initialSelectedAccounts[scope] = details.accounts[0];
+        }
+
```

Comment:
stupid nit, but can the `initialSelectedMethods[scope] = 'eth_blockNumber';` get moved below the setting of `initialSelectedAccounts` so that it's closer to the openRPCDoc lookup stuff?

Reply:
done here: https://github.com/MetaMask/test-dapp-multichain/pull/21/commits/39d9575c0d355a7836dc72352c457162e81f454f